pkg/config/datafileprojectconfig/config.go

```diff
@@ -27,6 +28,17 @@ import (
 
 var logger = logging.GetLogger("DatafileProjectConfig")
 
+var datafileVersions = []string{"4"}
```

Comment:
Consider using a `map[string]struct{}`, then you can check via `_, ok := datafileVersions[datafile.Version]` and we don't need the `inSlice` method defined either.

https://stackoverflow.com/questions/10485743/contains-method-for-a-slice